src/main/java/com/cloudbees/jenkins/Cleaner.java

```diff
@@ -29,70 +27,50 @@
  */
 @Extension
 public class Cleaner extends PeriodicWork {
-    private final Set<GitHubRepositoryName> couldHaveBeenRemoved = new HashSet<GitHubRepositoryName>();
+    /**
+     * Queue contains repo names prepared to cleanup.
+     * After configure method on job, trigger calls {@link #onStop(AbstractProject)}
+     * which converts to repo names with help of contributors.
+     *
```

Comment:
think nobody will be read this in generated javadoc format. Only here in code. So don't like `<p/>`